README.md

```diff
@@ -3,7 +3,8 @@
 
 The `vm-superio` crate provides emulation for legacy devices. For now, it offers
 this support only for the
-[Linux serial console](https://en.wikipedia.org/wiki/Linux_console).
+[Linux serial console](https://en.wikipedia.org/wiki/Linux_console) and a minimal
+[i8042 PS/2 Controller](https://web.archive.org/web/20200602023256/https://wiki.osdev.org/%228042%22_PS/2_Controller).
```

Comment:
This link looks odd. Does https://wiki.osdev.org/%228042%22_PS/2_Controller do the trick? Same for line 54.

Reply:
Yup, thanks.